test_package/CMakeUserPresets.json

```diff
@@ -0,0 +1,9 @@
+{
+    "version": 4,
+    "vendor": {
+        "conan": {}
+    },
+    "include": [
+        "build/apple-clang-15-armv8-20-release/generators/CMakePresets.json"
+    ]
+}
```

Comment:
Missing newline at EOF

Reply:
I still feel like this file should be removed altogether, although conan keeps generating it.

Reply:
This feels like something that needs to be excluded by `.gitignore`. It even references something from the build folder